bundle/CMakeLists.txt

```
@@ -97,3 +104,4 @@ install(
   DESTINATION bundle_conf
 )
 
+install (SCRIPT "${CMAKE_SOURCE_DIR}/PostInstall.cmake")

```

I think this should be a separate build step and not be done by default in `make` or `make install`

---

can we move this to a test similar to https://github.com/MeteoSwiss-APN/gtclang/blob/master/test/utils/gtclang-tester/CMakeLists.txt

---

I was forgetting the problem of having to do it at install-time....
```suggestion
if( DAWN_BUNDLE_PYTHON )
  install (SCRIPT "${CMAKE_SOURCE_DIR}/PostInstall.cmake")
endif()
```